### ISSUE TITLE
feature: item search

### ISSUE THREAD

**Issue by Bonkeybee** [Oppzippy/GlobalSearch#9]
Would it be possible to add an item search which queries for items in some internal database? An addon which did this for TBC was called "Finder"

example:
alt+space: "rich purp"
would show results like "rich purple silk shirt"

**Comment by Oppzippy** [comment]
To clarify, you mean you're looking to search every item in the game? Right now, only items with a use action in your bags are included.

If items not in your bags were included, what action should be performed when the item is selected? Or is the purpose of this just for seeing the tooltip and linking the item?

**Comment by Bonkeybee** [comment]
Right, it would mainly just be for item lookup to link. A common use case
would be to lookup an item to link to chat for tooltip info which would
have populated data from other addons like an auction house addon for price
data or bank data from bag addons.
